src/composables/queries/usePoolsQuery.ts

```diff
@@ -39,14 +39,14 @@ type FilterOptions = {
 export default function usePoolsQuery(
   filterTokens: Ref<string[]> = ref([]),
   options: UseInfiniteQueryOptions<PoolsQueryResponse> = {},
-  filterOptions?: FilterOptions
+  filterOptions?: FilterOptions,
+  poolsSortField?: Ref<string> | undefined
```

Comment:
Do you need `| undefined` when the param is optional?

Reply:
good catch, thanks!